### PR TITLE
Refactor error logs in enforcer

### LINKED ISSUE #2669: Refactor error logs in enforcer into formalized format
### Describe your problem(s)
Since we introduced a new formalized logging format for error logs in https://github.com/wso2/product-microgateway/issues/2422, we need to refactor some(important) error logs in enforcer as an initial step.

Going forward, with developing new features, we will be gradually converting all error logs into formalized form.

### Describe your solution
Use the following format for errors..
Ex:
```
logger.error("Test error msg", errorLog(LoggingConstants.Severity.CRITICAL, 349000));
```
#### Related Issues:
#2422

### PR DESCRIPTION
**Pull request by VajiraPrabuddhaka**
### Purpose
This PR will refactor some error log statements in enforcer, according to the formal logging introduced in https://github.com/wso2/product-microgateway/issues/2422

### Issues

Fixes #2669 

### Automation tests
 - Unit tests added: Yes/No
 - Integration tests added: Yes/No

### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [x] Validated respective github issues
- [x] Assigned milestone to the github issue(s)

### CODE REVIEW

File: enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/analytics/AccessLoggingService.java [hunk 1]
```diff
@@ -43,6 +44,8 @@
 import java.util.concurrent.LinkedBlockingQueue;
 import java.util.concurrent.TimeUnit;
 
+import static org.wso2.choreo.connect.enforcer.commons.logging.ErrorDetails.errorLog;
+
```
Comment (Rajith90) [inline enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/analytics/AccessLoggingService.java:48]: Lets try not to use the static import. There are other places as well in the same PR

File: enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/analytics/AccessLoggingService.java [hunk 2]
```diff
@@ -76,7 +79,8 @@ public void onNext(StreamAccessLogsMessage message) {
 
             @Override
             public void onError(Throwable throwable) {
-                logger.error("Error while receiving access log entries from router. " + throwable.getMessage());
+                logger.error("Error while receiving access log entries from router. " + throwable.getMessage(),
+                        errorLog(LoggingConstants.Severity.CRITICAL, 5101));
```
Comment (Rajith90) [inline enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/analytics/AccessLoggingService.java:83]: Is this error codes documented somewhere

Comment (VajiraPrabuddhaka) [reply enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/analytics/AccessLoggingService.java:83]: Yes, we are maintaining a list of error codes along with some description, separately for enforcer and adapter..

File: enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/analytics/AccessLoggingService.java (hunk 1 above)
Comment (VajiraPrabuddhaka) [reply enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/analytics/AccessLoggingService.java]: Fixed in https://github.com/wso2/product-microgateway/pull/2666/commits/51303937f8b421660770bc244d87f82ab1951668

**Review by pubudu538 — APPROVED** [review]

**Review by praminda — APPROVED** [review]

**Pull request merged**